src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.ts

```diff
@@ -83,6 +85,10 @@ export class GqrRqgViewResponsesComponent extends ResponsesInstructorCommentsBas
             this.teamsToUsers[response.giverTeam].push(response.giver);
             this.teamExpanded[response.giverTeam] = this.isExpandAll;
           }
+          this.userToEmail[response.giver] = this.userToEmail[response.giver] || '';
+          if (this.userToEmail[response.giver].indexOf(response.giver) === -1 && response.giverEmail !== undefined) {
```

Comment:
If you want to check something not to be undefined.
You can do
```suggestion
          if (this.userToEmail[response.giver].indexOf(response.giver) === -1 && response.giverEmail) {
```

Reply:
Same here, what the `indexOf` for?

Reply:
Initially was to check for undefined keys in the map but I forgot to remove it after adding the initialization. Have removed it now